pandas/core/tools/numeric.py

```diff
@@ -290,15 +290,17 @@ def to_numeric(
             klass = FloatingArray
         values = klass(data, mask)
 
-        if dtype_backend == "pyarrow" or isinstance(values_dtype, pd.ArrowDtype):
+        if dtype_backend == "pyarrow" or isinstance(values_dtype, ArrowDtype):
             values = ArrowExtensionArray(values.__arrow_array__())
 
     if is_series:
         return arg._constructor(values, index=arg.index, name=arg.name)
     elif is_index:
         # because we want to coerce to numeric if possible,
         # do not use _shallow_copy
-        return pd.Index(values, name=arg.name)
+        from pandas import Index
```

Comment:
If we can do `import pandas` in the top name space, we can also do `from pandas import Index` in the top namespace?

Reply:
the point of this change is to avoid importing from the top namespace at the module level

Reply:
Yes, I missed that the `import pandas as pd` line in the top is being removed is this PR also.